flexmeasures/cli/data_show.py

```diff
@@ -1,4 +1,6 @@
-"""CLI Tasks for listing database contents - most useful in development"""
+"""
+CLI commands for listing database contents.
```

Comment:
Not just database contents. For example reporters and schedulers aren't in the database (yet).